client/.eslintrc.js

```diff
@@ -13,5 +13,6 @@ module.exports = {
         usePrettierrc: true,
       },
     ],
+    '@next/next/no-html-link-for-pages': ['error', 'client/src/pages'],
```

Comment:
are you sure this is really needed?

Reply:
Yeah, so [this rule](https://nextjs.org/docs/messages/no-html-link-for-pages) works correctly. You can see the warning in the ESLint output of the editor

Reply:
![Screenshot 2022-06-29 at 09 40 00](https://user-images.githubusercontent.com/674179/176379944-7a6a493a-9855-4676-8398-388655b6d97a.png)
I'll remove it because I have the same warning with and without this rule. In case we need it, we can open a new PR only with this change.